modules/security-context-constraints-psa-opting.adoc

```diff
@@ -10,22 +10,24 @@ You can enable or disable automatic pod security admission synchronization for m
 
 [IMPORTANT]
 ====
-Namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
+Most namespaces that are defined as part of the cluster payload have pod security admission synchronization disabled permanently. These namespaces include:
 
 * `default`
 * `kube-node-lease`
 * `kube-system`
 * `kube-public`
 * `openshift`
-* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
+* System-created namespaces that are prefixed with `openshift-` and do not contain an Operator
 
-By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
+By default, namespaces that have an `openshift-` prefix with an Operator have pod security admission synchronization enabled. After a cluster service version (CSV) is created in an `openshift-*` namespace, pod security admission synchronization is turned on. The label inherits the permissions set in service account of the namespace.
```

Comment:
```suggestion
By default, namespaces that have an `openshift-` prefix with an Operator have pod security admission synchronization enabled. After a cluster service version (CSV) is created in an `openshift-*` namespace, pod security admission synchronization is turned on. The label inherits the permissions set in service account of the namespace.
```
NOTE: Reference to the changes introduced in 4.12 and plans for PSA label synchronization are in the following WIP AUTH release notes PR: https://github.com/openshift/openshift-docs/pull/52541/files

Reply:
> By default, namespaces that have an `openshift-` prefix with an Operator have pod security admission synchronization enabled.

By default, namespaces that have an `openshift-` prefix and have an Operator installed have pod security admission synchronization enabled.

> The label inherits the permissions set in service account of the namespace.

The label inherits the permissions of the service account(s) in the namespace.